### DISCUSSION TITLE
blog/2023-in-review

### DISCUSSION THREAD

**Discussion by TkDodo** [TkDodo/blog-comments#101] [Announcements]
A retrospective on what has happened for me in 2023

https://tkdodo.eu/blog/2023-in-review

**Comment by DanWeissman** [comment]
Looking forward to the course with the UI.dev guys - I know you said early 2024 - have you got a more solid date yet? :-)

**Reply by TkDodo** [reply]
I can't be too specific because we're aiming for the best quality, not a specific timeline. Things shift around constantly, blockers show up, and as I said, I only do this part-time. I am actively working on it, I don't want to drag it on for too long, but that's the best I can do at the moment :)

**Reply by DanWeissman** [reply]
No problem, thank's for the reply!
